openfecli/tests/commands/test_gather.py

```diff
@@ -145,3 +205,29 @@ def test_missing_leg_allow_partial(results_dir):
     result = runner.invoke(gather,
                            ['results'] + ['--allow-partial', '-o', '-'])
     assert result.exit_code == 0
+
+
+RBFE_RESULTS = pooch.create(
+    pooch.os_cache('openfe'),
+    base_url="doi:10.6084/m9.figshare.24542059",
+    registry={"results.tar.gz": None},
+)
+
+
+@pytest.fixture
+def rbfe_results():
+    # fetches rbfe results from online
+    # untars into local directory and returns path to this
+    d = RBFE_RESULTS.fetch('results.tar.gz', processor=pooch.Untar())
+
+    return os.path.join(pooch.os_cache('openfe'), 'results.tar.gz.untar', 'results')
+
+
+@pytest.mark.download
```

Comment:
we should probably also add in a pytest plugin that turns tests requiring downloads off by default?